ui/lesson/ScriptingChallenge/ScriptingChallenge.tsx

```diff
@@ -57,25 +76,26 @@ export default function ScriptingChallenge({
   }
 
   return (
-    <div className="grid grid-cols-2">
-      <div className="flex flex-col gap-4 border-r border-white border-opacity-30 p-10">
-        {children}
-      </div>
+    <Lesson
+      direction={
+        isSmallScreen ? LessonDirection.Vertical : LessonDirection.Horizontal
+      }
+    >
+      <LessonTabs items={tabData} classes="px-4 py-2 w-full" stretch={true} />
+      {children}
 
-      <div className="flex flex-col">
+      <div className="flex max-h-[92.5vh] max-w-full grow flex-col border-white/25 font-space-mono text-white md:max-w-[50%] md:basis-1/3 md:border-l">
```

Comment:
What's the reason for the 92.5% viewport height via `max-h-[92.5vh]`? It leaves a bit of empty space at the bottom (on desktop).

Do you need `max-w-full`?

Reply:
Good catch! They aren't necessary, these were here as an artifact from trying to get the terminal window to play nice